### PR TITLE
TG-28: Project model created

### PR DESCRIPTION
**Pull request by JuanMornik**

### CODE REVIEW

File: src/models/Projects.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+const { Schema } = mongoose;
+
+const projectSchema = new Schema(
+
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+
```
Comment (Javier-Cervera) [inline src/models/Projects.js:12]: Spaces between elements wouldn't be necessary.

**Review by Javier-Cervera — COMMENTED** [review]
Looks good to me

File: src/models/Projects.js (hunk 1 above)
Comment (Javier-Cervera) [inline src/models/Projects.js:12]: Spaces between elements wouldn't be necessary.

File: src/models/Projects.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    team: [
+      {
+        name: { type: String, required: true },
+        role: { type: String, required: true },
+        hours: { type: Number, required: true },
+        rate: { type: Number, required: true },
+      },
+    ],
```
Comment (SebastianRonchetti) [inline src/models/Projects.js:45]: Projects should have tasks assigned so perhaps a similar array could be made in a comment to express said link

File: src/models/Projects.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+    /* adminId: {
+      type: Number,
+      required: true,
+    }, */
```
Comment (AlfonsoDalix) [inline src/models/Projects.js:50]: Descamoentemos esto y le podemos pasar que sea de tipo String

File: src/models/Projects.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+    projectManager: {
+      type: String,
+      required: true,
+    },
+
+    team: [
+      {
+        name: { type: String, required: true },
+        role: { type: String, required: true },
```
Comment (AlfonsoDalix) [inline src/models/Projects.js:41]: el role lo podemos hacer de tipo string y le pasamos el enum de opciones como:
enum: ['DEV', 'QA', 'PM', 'TL']

File: src/models/Projects.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+      required: false,
+    },
+
+    projectManager: {
+      type: String,
+      required: true,
+    },
+
+    team: [
+      {
```
Comment (AlfonsoDalix) [inline src/models/Projects.js:39]: Deberíamos agregar tambien un id: usemos tipo String y required

**Review by AlfonsoDalix — CHANGES_REQUESTED** [review]

**Review by AlfonsoDalix — APPROVED** [review]

**Pull request merged**